docs/create_allocator_service.md

```diff
@@ -0,0 +1,435 @@
+
+## Objectives
+- Create a secure allocator service
+- Deploy the service to GKE
+- Allocate a Game Server from a Fleet using the service
+
+
+:warning: This quickstart requires Agones installed with agones.image.tag=0.5.0-03f4866 or newer :warning:
+
+## Prerequisites
+1. A [Go](https://golang.org/dl/) environment
+2. [Docker](https://www.docker.com/get-started/)
+3. Agones installed on GKE, with a simple-udp fleet
+4. kubectl properly configured
+5. A clone or fork of the [Agones repo](https://github.com/GoogleCloudPlatform/agones)
+
+
+>NOTE: Agones required Kubernetes versions 1.9 with role-based access controls (RBAC) and MutatingAdmissionWebhook features activated. To check your version, enter `kubectl version`.
+
+To install on GKE, follow the install instructions (if you haven't already) at
+[Setting up a Google Kubernetes Engine (GKE) cluster](../install/README.md#setting-up-a-google-kubernetes-engine-gke-cluster). Also complete the "Enabling creation of RBAC resources" and "Installing Agones" sets of instructions on the same page.
+
+While not required, you may wish to review the [Create a Game Server](../docs/create_gameserver.md), [Create Game Server Fleet](../docs/create_fleet.md), and/or [Edit a Game Server](../docs/edit_first_game_server.md) quickstarts.
+
+
+### 1. Create Firewall Rules
+
+Let's start off by making some firewall rules that will be used by kubernetes health checks and the ingress which we will create shortly.  
+
+First, we will make one for the HTTPS health checks that will be sent to our service by running:
+```
+gcloud compute firewall-rules create fleet-allocator-healthcheck \
+  --allow tcp \
+  --source-ranges 130.211.0.0/22,35.191.0.0/16 \
+  --target-tags fleet-allocator \
+  --description "Firewall to allow health check of fleet allocator service"
+```
+
+The output should be something like:
+```
+Creating firewall...done.                                                       
+NAME                         NETWORK  DIRECTION  PRIORITY  ALLOW  DENY  DISABLED
+fleet-allocator-healthcheck  default  INGRESS    1000      tcp          False
+```
+
+Create a firewall rule for nodePort traffic on the range of ports used by Ingress services of type nodePort.  We are using nobdePort becuase it supports TLS.
+```
+gcloud compute firewall-rules create nodeport-rule \
+  --allow=tcp:30000-32767 \
+  --target-tags fleet-allocator \
+  --description "Firewall to allow nodePort traffic of fleet allocator service"
+```
+
+The output should be something like:
+```
+Creating firewall...done.                                                       
+NAME           NETWORK  DIRECTION  PRIORITY  ALLOW            DENY  DISABLED
+nodeport-rule  default  INGRESS    1000      tcp:30000-32767        False
+```
+
+
+### 2. Make It Secure
```

Comment:
I would not worry about this for now. I think we can mention it (maybe in a "next steps section"?), but I think it will detract from the example we are trying to explain.

Reply:
Yes, some people will find this to be a pain and might just skip security all together.  But I think that if we describe how to do something, we would be remiss to not describe how to do it as safely as possible.